mmv1/third_party/terraform/services/pubsub/resource_pubsub_topic_test.go

```diff
@@ -207,6 +207,44 @@ func TestAccPubsubTopic_cloudStorageIngestionUpdate(t *testing.T) {
 	})
 }
 
+func TestAccPubsubTopic_messageStoragePolicyRemove(t *testing.T) {
+	resourceName := "google_pubsub_topic.usage-metrics"
+
+	initialConfig := `
+resource "google_pubsub_topic" "usage-metrics" {
+  name = "topic-name"
+
+  message_storage_policy {
+    allowed_persistence_regions = ["us-central1"]
+  }
+}
+`
+
+	updatedConfig := `
+resource "google_pubsub_topic" "usage-metrics" {
+  name = "topic-name"
```

Comment:
this isn't actually testing the change on `allowedPersistenceRegions`, this is testing removing `message_storage_policy`